webpack.dev.js

```diff
@@ -48,17 +48,14 @@ module.exports = merge.smart(require('./webpack.base.js'), {
     }),
     new webpack.HotModuleReplacementPlugin(),
     new webpack.NoEmitOnErrorsPlugin(),
-    new ExtractTextPlugin({
-      filename: '[name].css',
-      disable: true,
```

Comment:
Weird that this was even there, good catch. Wondering if we even need it at all for development, might be faster without. Not sure.